fastlane/metadata/fr-FR/release_notes.txt

```diff
@@ -1,4 +1,4 @@
-C’est le moment de dépoussiérer votre collection de fichiers .gif : les notifications et les commentaires prennent désormais en charge vos fichiers animés préférés ! (Malheureusement, cette mise à jour ne règle toujours pas la question de savoir si l'on prononce "gif" ou "jif").
+C’est le moment de dépoussiérer votre collection de fichiers .gif : les notifications et les commentaires prennent désormais en charge vos fichiers animés préférés ! (Malheureusement, cette mise à jour ne règle toujours pas la question de savoir si l’on prononce « guif » ou « jif ».)
```

Comment:
Interesting change... 🤔 The validation to upload to ASC passed and the notes are sort of transient, so I don't feel like it's worth following up on whether curly/smart apostrophes and quotes are better than straight alternative 🤷‍♂️